src/lib/cdn.js

```diff
@@ -0,0 +1,201 @@
+/******************************************************************************
+Copyright (c) 2011, Google Inc.
+All rights reserved.
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+    * Redistributions of source code must retain the above copyright notice,
+      this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright notice,
```

Comment:
This is the copyright notice we're required to include from the source of this data:

https://github.com/mozilla/pulse/pull/94/files/206d356bb31599aeaf3961957a596c1b70bc548b#diff-96c00a95c14891d2849c138a9802caa0R27